packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -515,3 +542,31 @@ class ErrorNavigator extends StatelessWidget {
     onSelectError(newIndex);
   }
 }
+
+class PubRootDirectorySection extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    preferences.inspector.loadCustomPubRootDirectories();
+    return ValueListenableBuilder<List<IsolateRef?>>(
+      valueListenable: serviceManager.isolateManager.isolates,
```

Comment:
same question wrt whether we should be listening to the selected isolate or to all isolates